app/services/force/application_service.rb

```diff
@@ -45,6 +45,7 @@ def lease(application_id)
              .select(:Id,
                      :Unit__c,
                      :Lease_Start_Date__c,
+                     :Lease_Status__c,
```

Comment:
move this to a separate service too. Lease Service.

Combine services in the controller